.github/workflows/ci.yml

```diff
@@ -330,12 +331,51 @@ jobs:
           exit ${return}
       - name: Test
         run: |
-          (for DIM in 2 3; do \
-            printf "\n-------- Testing ${DIM}D --------\n"; \
-            cd ${{runner.workspace}}/build-${DIM}d-${{matrix.os}}-${{matrix.build_type}}; \
-            ctest ${{matrix.ctest_args}} -VV --output-on-failure; \
-            if [ $? -ne 0 ]; then exit 1; fi \
-          done)
+          cd ${{runner.workspace}}/build-${{matrix.dim}}d-${{matrix.os}}-${{matrix.build_type}}; \
+          ctest ${{matrix.ctest_args}} -VV --output-on-failure; \
+          if [ $? -ne 0 ]; then exit 1; fi \
+      - name: Checkout Reference Version
+        if: env.DO_REF_COMPARE == 'true'
+        run : |
+          git remote add amrexcomb https://github.com/AMReX-Combustion/PeleC.git
```

Comment:
I doubt it's a big deal, but I would probably use the checkout action here if possible.

Reply:
Not sure if the checkout action can be used here because we need to fetch from a different remote. Instead, I just specified a shallow fetch and submodule update to reduce the download sizes. I made all the other recommended changes.